### DISCUSSION TITLE
I would like to propose a section.

### DISCUSSION THREAD

**Discussion by geologistluna** [CaptainBobSim/The-Cessna-172-Project-V3#228] [Ideas]
Consider putting into here, a list of all electronic parts that are required.  Such parts would include the type of switches you have designed the panels for, with full proper names and even order codes if possible.  The names and types of stepper motors required, I know you have two different types for the instruments, and I have found them. but others may not find them so easily, OR you may not find the ones you are looking for if you don't have a common place to store this information.  

As an example, I've now purchased two different types of SPST switches that do not fit into the analog switch panel, and the original electromagnets you have listed are no longer even available.  I don't wish to buy a ton of EM magnets that may or may not work for the panel.  

Please let me know if I can help you organize this in any way.

**Comment by CaptainBobSim** [comment]
Hello!
I agree - this would be a great asset to the project.
I usually try to give rough guidelines in the README files, but as you pointed out there are variations of components.

Most details are in the descriptions of the YouTube videos, and if it matches what you see in the video it's usually good enough.
I'm currently working on the EM situation - I'm designing a modification for a different EM.

I don't know if I see your vision fully, could you make a component template based on what you'd like to see?
I also struggle with finding parts with DigiKey and Mouser, so any help would be appreciated there.

**Reply by geologistluna** [reply]
Let me look into a BOM type system where we build a single instrument and include all parts to it.  I will work on a format for this.

**Reply by CaptainBobSim** [reply]
Sounds great!
